flow/project.py

```diff
@@ -3549,9 +3563,9 @@ def submit(
         ----------
         bundle_size : int
             Specify the number of operations to be bundled into one submission, defaults to 1.
-        jobs : iterable of :class:`~signac.contrib.job.Job`
-            Only submit operations for the given jobs, or all if the argument
-            is None. (Default value = None)
+        jobs : iterable of :class:`~signac.contrib.job.Job` or aggregates of jobs
+            Only submit operations for the given jobs or aggregates of jobs,
+            or all if the argument is None. (Default value = None)
```

Comment:
```suggestion
            or all the associated aggregates/jobs for an operation if the argument is None. (Default value = None)
```

This document really raises the question that do we want our users to know that internally, a normal operation always acts on aggregate of single job?

Reply:
I think the suggestion is okay but not necessary. Either way is fine with me. Maybe get another opinion on this.

Reply:
Resolved by consensus on a new wording.